File: VB/Q240157/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace Q240157
	Partial Public Class Form1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
			For i As Integer = 0 To 9
				dataTable1.Rows.Add(i Mod 2 = 0, i Mod 2 = 1)
			Next i
		End Sub
	End Class
End Namespace

File: CS/Q240157/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace Q240157 {
    public partial class Form1 : XtraForm {
        public Form1() {
            InitializeComponent();
            for (int i = 0; i < 10; i++)
                dataTable1.Rows.Add(i % 2 == 0, i % 2 == 1);
        }
    }
}